hack/build-rpm-release.sh

```diff
@@ -1 +0,0 @@
-build-rpms.sh
```

Comment:
Why did you change that? This supposed to be link to `hack/build-rpms.sh`?

Reply:
Even more the two are not different after your pr.

Reply:
accident during rebase